api/lib/gateways/Academy-Benefits/FetchCustomerAPI.js

```diff
@@ -0,0 +1,64 @@
+const rp = require('request-promise');
+const { Systems } = require('../../Constants');
+const {
+  checkString,
+  nameCase,
+  formatAddress,
+  formatRecordDate,
+  upperCase
+} = require('../../Utils');
+
+module.exports = options => {
+  const baseUrl = options.baseUrl;
+  const apiKey = options.apiKey;
+
+  let processCustomer = function(record) {
```

Comment:
processCustomer (here) and processRecord (in FetchCustomerDB) seem to perform the same function. Could you extract the function to reduce repeating the code?

Reply:
That's a good suggestion, I could look into refactoring the code there